tests/integration/test_pipe_simple.py

```diff
@@ -60,37 +61,40 @@ def outputs(self) -> List[str]:
         return ["response"]
 
 
-def test_pipeline_cached():
-    def run_pipeline():
-        with Pipeline() as pipeline:
-            load_hub_dataset = LoadHubDataset(name="load_dataset", batch_size=8)
-            rename_columns = RenameColumns(name="rename_columns", input_batch_size=12)
-            generate_response = GenerateResponse(
-                name="generate_response", input_batch_size=16
-            )
-
-            load_hub_dataset.connect(rename_columns)
-            rename_columns.connect(generate_response)
-
-            pipeline.run(
-                parameters={
-                    "load_dataset": {
-                        "repo_id": "plaguss/test",
-                        "split": "train",
+def run_pipeline():
+    with Pipeline() as pipeline:
+        load_hub_dataset = LoadHubDataset(name="load_dataset", batch_size=8)
+        rename_columns = RenameColumns(name="rename_columns", input_batch_size=12)
+        generate_response = GenerateResponse(
+            name="generate_response", input_batch_size=16
+        )
+
+        load_hub_dataset.connect(rename_columns)
+        rename_columns.connect(generate_response)
+
+        return pipeline.run(
+            parameters={
+                "load_dataset": {
+                    "repo_id": "plaguss/test",
```

Comment:
Unrelated to this PR, but maybe it's time to create `argilla-internal-testing` in Hugging Face Hub? 😆

Reply:
yes... the current workflow is weird at best 😆